.github/workflows/web.yml

```diff
@@ -55,17 +55,21 @@ jobs:
       - uses: actions/checkout@v3
         with:
           fetch-depth: 0
-      - uses: actions-rs/toolchain@v1
-        with:
-          toolchain: stable
-          override: true
+      - uses: dtolnay/rust-toolchain@stable
+      - uses: swatinem/rust-cache@v2
+      
+      # Install mdbook plugins
+      - name: Install mdbook plugins
+        run: |
+          cargo install --locked mdbook-toc@0.11.2
```

Comment:
oranda doesn't use these mdbook yet so we should remove them until it does i think